### ISSUE TITLE
How can I use post_bootstrap or post_init ?

### ISSUE THREAD

**Issue by arezvani** [zalando/postgres-operator#2309]
I want to create user with specific permission. For example I don't want the user that I create be the admin ( member of admin role) and just have access to specific database not postgres.

So if I can use post_bootstrap or post_init, my problem get solved.

**Comment by FxKu** [comment]
Have you already looked into what prepared databases can provide you with?
https://github.com/zalando/postgres-operator/blob/master/docs/user.md#prepared-databases-with-roles-and-default-privileges

**Comment by arezvani** [comment]
Yes but it doesn't solve my problem because I want to execute script after database and user creation for example change user grants or populate database.

**Comment by FxKu** [comment]
Use a tool to run SQL migrations like flyway etc.. We do not plan to extend the operator to run arbitrary SQL script. By creating schemas and extensions for prepared databases we might have even went too far :smiley:

**Comment by arezvani** [comment]
@FxKu Thanks.
You don't have any plan for user management with CRD?
For example after database gets ready, I can create users with different roles and grant like read only or read write user. With that I can list my users with their roles and privileges with CRD.

**Comment by FxKu** [comment]
Currently no plan for such a CRD. But in general, a CRD sounds indeed like a way to go. We already have the PostgresTeam CRD for assigning more teams and single members to a database cluster. So user management could be solved in a similar fashion. But such stuff should also be possible with schema migration tools, no?
